src/SAWScript/Interpreter.hs

```diff
@@ -2616,6 +2634,16 @@ primitives = Map.fromList
     Current
     [ "Use the default set of callee-saved registers during x86 verification." ]
 
+  , prim "enable_what4_eval" "TopLevel ()"
+    (pureVal enable_what4_eval)
+    Experimental
+    [ "Enable What4 translation for SAwCore expressions during Crucible symbolic execution." ]
```

Comment:
Minor typo here, and below "SAwCore", should have some consistent capitalization.